domains/Accounts/Tests/Unit/UserModelTest.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+namespace Domains\Accounts\Tests\Unit;
+
+use Carbon\Carbon;
+use Domains\Accounts\Database\Factories\UserFactory;
+use Domains\Accounts\Models\User;
+use Illuminate\Database\Eloquent\SoftDeletingScope;
+use Tests\TestCase;
+
+class UserModelTest extends TestCase
+{
+    private User $model;
+
+    protected function setUp(): void
+    {
+        parent::setUp();
+
+        $this->model = UserFactory::new()->unverified()->make();
+    }
+
+    /** @test */
+    public function it_contains_required_properties(): void
+    {
+        self::assertIsString($this->model->name);
+        self::assertIsString($this->model->email);
+        self::assertIsString($this->model->password);
+        self::assertNull($this->model->email_verified_at);
+        self::assertIsBool($this->model->trusted);
```

Comment:
We should assert the default value of `trusted` is `false`.

Reply:
yep